src/block/utils/ComposterUtils.php

```diff
@@ -0,0 +1,261 @@
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 9), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 10), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 11), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 12), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 13), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::BROWN_MUSHROOM_BLOCK, 14), VanillaBlocks::BROWN_MUSHROOM_BLOCK()), 85);
+
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 0), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 1), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 2), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 3), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 4), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 5), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 6), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 7), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 8), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 9), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 10), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 11), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 12), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 13), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+		self::register(new ItemBlock(new ItemIdentifier(ItemIds::RED_MUSHROOM_BLOCK, 14), VanillaBlocks::RED_MUSHROOM_BLOCK()), 85);
+
+		//TODO: add Nether Wart Block, Warped Wart Blocks
+
+		// region 100% percentage compost
+		self::register(new Item(new ItemIdentifier(ItemIds::CAKE, 0), "Cake"), 100);
+		self::register(new Item(new ItemIdentifier(ItemIds::PUMPKIN_PIE, 0), "Pumpkin Pie"), 100);
+	}
```

Comment:
You're not supposed to use `new Item`. Use `VanillaBlocks` and `VanillaItems`.

Reply:
> You're not supposed to use `new Item`. Use `VanillaBlocks` and `VanillaItems`.

yeah, i'm sorry